### ISSUE TITLE
A11y_AzureToolsStorageExplorer_RenameBlobContainer_ScreenReader: NVDA does not read the alert message while NVDA focus goes to the edit box.

### ISSUE THREAD

**Issue by Radha2019** [microsoft/AzureStorageExplorer#1882]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1;
 #NVDA;  #Win10;

**Environment Details:**
Desktop App: Windows Version: Windows10
NVDA Version: 2019.1.1

**Steps to Reproduce:**
1. Launch Storage Explorer
2. Navigate to "Add Account" menu item present on left side of screen and hit enter.
3. Select "Add an Azure Account" radio button.
4. And Select "Azure" from "Azure Environment" combo box and click on Next button.
5. Sign in to Azure.
6. After having connected to Azure, go to "Manage Accounts" pane and hit enter.
7. Select the subscriptions you will use. Once selected, click on "Apply" button.
8. Toggle Explorer pane gets opened in left.
9. Navigate to a Storage account and expand it.
10. Navigate to "Blob Containers" and right click on it.
11. Select "Create Blob container" and Create a new Blob container.
12. Once created, navigate to the newly created blob container and right click on it.
13. Select "Rename" and hit enter.
18. "Rename Blob Container" Popup gets opened.
19. Open narrator and check whether the narrator reads the alert message while the focus goes to the Please Specify the new name.

**Actual:**
While navigating through keyboard, NVDA does not read the alert message while NVDA focus goes to the edit box.

**Same issue is observed throughout the application for all the alert message while NVDA focus goes to the edit box.**

**Expected:** 
NVDA should read the alert message after read the Please Specify the new name.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**User Impact:** 
Screen reader user unable to know that which dialog is opened.

**MAS Reference:**
MAS1.3.1:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
**Attachment for Reference**
[1882_A11y_AzureToolsStorageExplorer_RenameBlobContainer_ScreenReader.pptx](https://github.com/microsoft/AzureStorageExplorer/files/3662066/1882_A11y_AzureToolsStorageExplorer_RenameBlobContainer_ScreenReader.pptx)

**Comment by MRayermannMSFT** [comment]
Look into aria-described by.

**Comment by MRayermannMSFT** [comment]
Related: https://github.com/microsoft/AzureStorageExplorer/issues/1958

**Comment by MRayermannMSFT** [comment]
Fix checked into master for 1.12

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by vinodbk197** [comment]
Verified this issue on:Azure Tools Storage Explorer (Version: 1.12.0)
The issue is no longer to reproduces, Hence closing this bug.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Win32-Sep2019;
